### PR TITLE
dgen data config files

### PR DESCRIPTION
**Pull request by ashreeta**
dataset and dataset dimension descriptions for dgen dpv generation

### CODE REVIEW

File: project/datasets/modeled/dgen/dimension_mappings/county_to_county.csv [hunk 1]
```diff
@@ -0,0 +1,3144 @@
...
+886,37005
+1016,42003
+787,39003
+3004,18003
+497,20001
+385,21003
+1096,22003
+462,45005
+2282,26007
+940,6003
```
Comment (elainethale) [inline project/datasets/modeled/dgen/dimension_mappings/county_to_county.csv:55]: The leading zeros in the FIPS codes are getting dropped (probably by Excel).

File: project/datasets/modeled/dgen/dimensions/dgen_county_fips_mapping.csv [hunk 2]
```diff
@@ -0,0 +1,3144 @@
+state_fips,county_fips,geoid10,county,state_abbr,state,county_id
+45,1,45001,Abbeville,SC,South Carolina,1210
+22,1,22001,Acadia,LA,Louisiana,2683
+51,1,51001,Accomack,VA,Virginia,467
+16,1,16001,Ada,ID,Idaho,2079
+40,1,40001,Adair,OK,Oklahoma,317
+21,1,21001,Adair,KY,Kentucky,1943
+19,1,19001,Adair,IA,Iowa,2306
+29,1,29001,Adair,MO,Missouri,3136
+8,1,8001,Adams,CO,Colorado,1188
```
Comment (elainethale) [inline project/datasets/modeled/dgen/dimensions/dgen_county_fips_mapping.csv:10]: Leading zeros of FIPS codes are getting dropped

File: project/datasets/modeled/dgen/dataset.json5 [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  data_schema: {
+    load_data_column_dimension: "metric",
+  },
+  dimensions: [
+    { 
+     "class": "County",
+      type: "geography",
+      name: "US_county_FIPS",
+      display_name: "County",
+      file: "dimensions/gen_county_fips_mapping.csv",
```
Comment (elainethale) [inline project/datasets/modeled/dgen/dataset.json5:40]: This filename is missing its first letter, i.e., "d".

More importantly, the file is not formatted in a way dsgrid can read--the headings need to be "id" and "name", and you probably need to include the time zone. See, e.g., https://github.com/dsgrid/dsgrid-project-StandardScenarios/blob/main/dsgrid_project/datasets/modeled/resstock/dimensions/conus_2022-resstock_geography_county_fips.csv

File: project/datasets/modeled/dgen/dimensions/sectors.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+id,name
+res,Residential
+com,Commercial
```
Comment (elainethale) [inline project/datasets/modeled/dgen/dimensions/sectors.csv:3]: I was expecting res and non-res. Either way is fine, just be consistent here and in dataset.json5 and in the data files.

**Review by elainethale — CHANGES_REQUESTED** [review]
Thanks for getting this started, @ashreeta!

File: project/datasets/modeled/dgen/dataset.json5 [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+      display_name: "res and non-res",
+      file: "dimensions/sectors.csv",
+      module: "dsgrid.dimension.standard",
+      "class": "Sector",
+      description: "Residential and non-residential sectors DPV adoption",
+    },
+    {
+      type: "metric",
+      name: "dpv_generation_kwh",
+      display_name: "dGen DPV generation",
```
[the remark this answers is not in the document]
Comment (ashreeta) [reply project/datasets/modeled/dgen/dataset.json5:56]: modified this, but am confused. Is there some guidance on what the name vs display name should be and what is the format? I see some inconsistency in how this is defined for tempo vs resstock/comstock.

File: project/datasets/modeled/dgen/dataset.json5 (hunk 3 above)
Comment (ashreeta) [reply project/datasets/modeled/dgen/dataset.json5:40]: modified this, but am confused. Is there more concrete guidance on the format? I see some inconsistency in how this is defined for tempo vs resstock/comstock.

**Pull request closed without merging**